crates/oxc_transformer/src/react_jsx/mod.rs

```diff
@@ -467,4 +455,53 @@ impl<'a> ReactJsx<'a> {
             }
         }
     }
+
+    fn transform_jsx_text(&self, text: &JSXText) -> Option<Expression<'a>> {
```

Comment:
We also need https://github.com/microsoft/TypeScript/blob/514f7e639a2a8466c075c766ee9857a30ed4e196/src/compiler/transformers/jsx.ts#L572